packages/jest-reporters/src/verbose_reporter.ts

```diff
@@ -137,17 +137,29 @@ export default class VerboseReporter extends DefaultReporter {
         );
       }
 
-      if (summedTests.todo > 0) {
-        this._logSummedTests(
-          'todo',
-          this._getIcon('todo'),
-          summedTests.todo,
-          indentLevel,
-        );
+      if (summedTests.todo.length > 0) {
+        summedTests.todo.forEach(todoTest => {
+          this._logTodoTest(
+            'todo',
+            this._getIcon('todo'),
```

Comment:
seems silly to pass in the icons and title, but I see it's consistent

Reply:
Yup. Didn't want to stray too far from what was being done. The same could be said for printing skipped tests

Reply:
I think given that we are explicitly pulling this behaviour out into `_logTodoTest` we should just move the concrete data into the function definition.

```ts
 private _logTodoTest(
    todoTest: TestResult.AssertionResult,
    indentLevel: number,
  ) {
    const text = chalk.dim(`todo ${todoTest.title}`);
    this._logLine(`${this._getIcon('todo')} ${text}`, indentLevel);
  }
```

Then if you want to be fancy you could curry this to:

```ts
 private _logTodoTest = (indentLevel: number) => (todoTest: TestResult.AssertionResult) => {
    const text = chalk.dim(`todo ${todoTest.title}`);
    this._logLine(`${this._getIcon('todo')} ${text}`, indentLevel);
  }
```

and then you can just do the following in the `forEach`:

```ts
summedTests.todo.forEach(this._logTodoTest(indentLevel));
```

Reply:
I like this approach. Should we do a similar approach to skipped tests then since their functions are very similar?